test/functionalTest/cases/0876_attribute_dates/notify_attr_dates_with_metadata.test

```diff
@@ -546,7 +546,7 @@ Date: REGEX(.*)
 =======================================================================
 POST http://localhost:REGEX(\d+)/notify
 Fiware-Servicepath: /
-Content-Length: 260
```

Comment:
The change in the lenght is due this test previously include duplicated keys, now removed. It seems that mjson.tool doesn't detect that and print one of the keys (I think the last one it gets in the input) in the pretty printed output.

Reply:
NTC